source/agent/quic/connection.js

```diff
@@ -0,0 +1,307 @@
+// MIT License
```

Comment:
Do you need this file in quic agent?

Reply:
Looks like it is not needed. `quicTransportStreamPipeline.js` plays the same role. I'll check it.